### PR TITLE
Allow oecert to not have output file

### PR DESCRIPTION
**Pull request by qiucwang**
Gave oecert option not to generate report/evidence/certificate output file

Signed-off-by: Qiucheng Wang <qiucwang@microsoft.com>

### CODE REVIEW

**Comment by yentsanglee** [comment]
Looks good. Please update README.md (the "out.bin").

**Review by ryanhsu19 — APPROVED** [review]

File: tests/tools/oecert/README.md [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
+
+Usage: `oecert Options`
+
+where `Options` are:
 
     --cert PRIVKEY PUBKEY : generate der remote attestation certificate.
     --report              : generate binary OE report.
     --evidence            : generate binary OE evidence.
-    --out FILENAME        : specify certificate/report/evidence output filename, default: out.bin
+    --out FILENAME        : specify certificate/report/evidence output filename.
```
Comment (yentsanglee) [inline tests/tools/oecert/README.md:21]: Tell user that there is no output if FILENAME not specified.

File: tests/tools/oecert/README.md [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
     --endorsements        : specify endorsements output filename.
     --verify              : verify generated certificate/report/evidence
     --log LOG_FILENAME    : log file name, default: oecert.log
     --verbose             : dump verbose info of evidence
 
-Example 1 Generate, verify and dump a ceritificate, output certificate to default output file "out.bin":
+Example 1 Generate, verify and dump a ceritificate:
 
     ./oecert --cert keyecec.pem publickeyec.pem --verify --verbose
```
Comment (yentsanglee) [inline tests/tools/oecert/README.md:29]: Without "--out", there will be no certificate output file.

File: tests/tools/oecert/README.md [hunk 3]
```diff
@@ -1,33 +1,34 @@
...
+Usage: `oecert Options`
+
+where `Options` are:
 
     --cert PRIVKEY PUBKEY : generate der remote attestation certificate.
     --report              : generate binary OE report.
     --evidence            : generate binary OE evidence.
-    --out FILENAME        : specify certificate/report/evidence output filename, default: out.bin
+    --out FILENAME        : specify certificate/report/evidence output filename.
     --endorsements        : specify endorsements output filename.
```
Comment (yentsanglee) [inline tests/tools/oecert/README.md:22]: Add ",default: null (no endorsements output)."

Comment (qiucwang) [reply tests/tools/oecert/README.md]: Added.

File: tests/tools/oecert/README.md (hunk 2 above)
Comment (qiucwang) [reply tests/tools/oecert/README.md:29]: Added.

**Review by mingweishih — APPROVED** [review]

**Review by yentsanglee — APPROVED** [review]

**Comment by yentsanglee** [comment]
bors try

**Pull request merged**